src/components/headers/index.js

```diff
@@ -0,0 +1,15 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Header from './item';
+
+export default class HeaderContainer extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return(
+    	<Header activeSubreddit={this.props.activeSubreddit} />
```

Comment:
I probably wouldn't create another component for this, it seems like you should be fine with just this component